.github/scripts/main/main.sh

```diff
@@ -129,7 +129,7 @@ prepare_project () {
     git checkout -b "$PR_BRANCH" "origin/$PR_BRANCH"
   fi
 
-  test -d _opam || opam switch create . --no-install --formula '"ocaml-system"'
+  test -d _opam || opam switch create . --no-install
```

Comment:
Why ? there is no ocaml installed ?

Reply:
Some testing slipped though, my bad, reverting